library/src/main/java/io/github/boguszpawlowski/composecalendar/week/WeekUtil.kt

```diff
@@ -48,3 +49,21 @@ internal fun YearMonth.getWeeks(
     )
   }
 }
+
+internal fun LocalDate.getWeek(
+  today: LocalDate = LocalDate.now(),
+): Week {
+  val firstDayOfWeek = minusDays(this.dayOfWeek.value.toLong() - 1)
```

Comment:
Shouldn't this take into account the `firstDayOfTheWeek` parameter?

Reply:
I'm sorry... I'll fix it. I will definitely add this option.